Aufgabe 3 c) + Zusatzt + Helfer

CynthiaN01 opened the pull request:
Funktionen für Aufgabe 3c) sind fertig und eine Zusatztfunktion ist dabei.

Guckt bitte mal drüber, auch für Tippfehler oder anderes.
Mir ist aufgefallen, dass ich nominal und nicht qualitativ geschrieben habe, soll ich das ändern?

Ihr könnt die Funktionen hiermit ausprobieren:
`# c)`
`# person = read.csv2("person.csv")`
`relate_categorial(person$Studienfach, person$Mathe.LK, dnames = c(NA, "LK"))`
`# Kontingenzmasse: 0.3580034 und 0.4766675`

`# Zusatz`
`rangkorr_koeff(person$Interesse.an.Mathematik, person$Interesse.an.Programmieren)`
`#[1] -0.5543833`
Ihr könnt auch die Eingabeparameter ändern und weiteres ausprobieren.

Zum mergen muss ich wahrscheinlich zuerst den fehlenden Inhalt aus dem main kopieren, mache ich dann wenn es soweit ist.

CynthiaN01 committed:
Merge pull request #10 from smtinova/main

Aktualisierung von branch Cynthia

CynthiaN01 committed:
Aufgabe 3c)

code für Aufgabe 3 c) und für seine Helferfunktion fertig.

CynthiaN01 committed:
Update Helferfunktion bei quad_Kontingenz

Änderung in den Formulierungen

CynthiaN01 committed:
Zusatzfunktion und sein Helfer

_eine Zusatzfunktion implementiert, die den Rangkorrelationskoeffizienten berechnet und zurück gibt.
_eine Helferfunktion, die Ränge berechnet nach beliebiger Anordnung.

Hanjin99 commented:
Ja, bitte änder nominal zu qualitativ, damit es einheitlich mit b ist. 
Es sieht gut aus.

CynthiaN01 commented:
ok, mache ich

CynthiaN01 committed:
3c) Bezeichnungen ändern

Das Argument nominal habe ich durch qualitativ ersetzt, damit es einheitlich ist.

CynthiaN01 commented:
Wenn es keine weiteren Änderungswünsche gibt, probiere ich morgen zu mergen mit Hanjins Idee.

CynthiaN01 committed:
Merge branch 'main' into Cynthia

CynthiaN01 committed:
Konflikt mit main lösen

Hier sind nochmal alle meine Änderungen und auch so, dass es keinen Konflikt mehr mit main gibt.
letztes Update vor dem merge
